packages/application-shell/src/components/quick-access/messages.ts

```diff
@@ -158,6 +158,10 @@ export default defineMessages({
     id: 'QuickAccess.openProjectSettingsChannelsTab',
     defaultMessage: 'Open Project Settings • Channels',
   },
+  openProjectSettingsStoresTab: {
+    id: 'QuickAccess.openProjectSettingsStoresTab',
+    defaultMessage: 'Open Project Settings • Stores',
+  },
```

Comment:
Will this be translated?

Reply:
I was misled by the different naming of the command compare to MC-FE and hence missed. It pushed them here https://github.com/commercetools/merchant-center-application-kit/pull/2079/commits/172a016d5d261398caf4bdd681f0bc01ffba486d